packages/flutter_tools/lib/src/cache.dart

```diff
@@ -220,7 +220,11 @@ class FlutterEngine {
   // Return a list of (cache directory path, download URL path) tuples.
   List<List<String>> _getToolsDirs() {
     if (Platform.isMacOS)
-      return <List<String>>[<String>['mac_debug', 'mac_debug/artifacts.zip']];
+      return <List<String>>[
+        <String>['darwin-x64', 'darwin-x64/artifacts.zip'],
+        <String>['android-arm-profile/darwin-x64', 'android-arm-profile/darwin-x64.zip'],
```

Comment:
Confused as to why we have this additional layer of mapping from bucket names to on-disk names.  But it's OK too.